### PR TITLE
Fix lint errors

### PR DESCRIPTION
**Pull request by jzohrab**
Replacement for PR #139.  Thanks for your patience with all of the thrash on that one.  This PR contains minimal changes.  Lint is still failing but I haven't introduced any extraneous changes.

### CODE REVIEW

File: org-super-agenda.el [hunk 1]
```diff
@@ -505,14 +505,18 @@ COMPARISON should be a symbol, one of: `past' or `before',
...
   (declare (indent defun))
   `(org-super-agenda--defgroup ,(intern (concat "effort" (symbol-name name)))
      ,(concat docstring "\nArgument is a time-duration string, like \"5\" or \"0:05\" for 5 minutes.")
      :section-name (concat "Effort " ,(symbol-name name) " "
                            (s-join " or " args) " items")
-     :let* ((effort-minutes (org-duration-string-to-minutes (car args))))
+     :let* ((effort-minutes (org-duration-to-minutes (car args))))
      :test (when-let ((item-effort (org-find-text-property-in-string 'effort item)))
-             (,comparator (org-duration-string-to-minutes item-effort) effort-minutes))))
+             (,comparator (org-duration-to-minutes item-effort) effort-minutes))))
```
Comment (alphapapa) [inline org-super-agenda.el:519]: Please remove these function name change hunks for now.  They may break compatibility with older versions of Org, which I'm not ready to deal with.

File: org-super-agenda.el [hunk 2]
```diff
@@ -713,6 +717,10 @@ e.g. \"A\" or (\"B\" \"C\")."
   :test (cl-member (org-super-agenda--get-priority-cookie item) args :test 'string=))
 
 (cl-defmacro org-super-agenda--defpriority-group (name docstring &key comparator)
+  "Define a priority-grouping function.
+NAME is a symbol appended to `priority' to create the group name.
+DOCSTRING is a string used for the function's docstring.
+COMPARATOR is a grouping comparison operator."
```
Comment (alphapapa) [inline org-super-agenda.el:723]: This should probably say, `COMPARATOR is an unquoted comparator function symbol.`

File: org-super-agenda.el [hunk 3]
```diff
@@ -713,6 +717,10 @@ e.g. \"A\" or (\"B\" \"C\")."
   :test (cl-member (org-super-agenda--get-priority-cookie item) args :test 'string=))
 
 (cl-defmacro org-super-agenda--defpriority-group (name docstring &key comparator)
+  "Define a priority-grouping function.
+NAME is a symbol appended to `priority' to create the group name.
+DOCSTRING is a string used for the function's docstring.
```
Comment (alphapapa) [inline org-super-agenda.el:722]: This should probably say, `DOCSTRING is the group's docstring.`

File: org-super-agenda.el [hunk 4]
```diff
@@ -713,6 +717,10 @@ e.g. \"A\" or (\"B\" \"C\")."
   :test (cl-member (org-super-agenda--get-priority-cookie item) args :test 'string=))
 
 (cl-defmacro org-super-agenda--defpriority-group (name docstring &key comparator)
+  "Define a priority-grouping function.
+NAME is a symbol appended to `priority' to create the group name.
```
Comment (alphapapa) [inline org-super-agenda.el:721]: Let's use:

```
The group is named priorityNAME.
```

File: org-super-agenda.el (hunk 2 above)
Comment (alphapapa) [inline org-super-agenda.el:723]: Then these sentences can be wrapped together, since they're so short.

**Review by alphapapa — CHANGES_REQUESTED** [review]
Thanks.

**Comment by jzohrab** [comment]
Hi @alphapapa, Unfortunately I'm not going to be able to make these changes for a long time.  Am currently travelling with no computer.  This doesn't seem critical so perhaps just close the PR.  Cheers, jz

**Pull request closed without merging**